core/admin/EE_Admin.core.php

```diff
@@ -8,6 +8,7 @@
 use EventEspresso\core\services\container\exceptions\ServiceNotFoundException;
 use EventEspresso\core\services\loaders\LoaderFactory;
 use EventEspresso\core\services\notifications\PersistentAdminNoticeManager;
+use WPStaging\Utils\Loader;
```

Comment:
oops... think you grabbed the wrong Loader class  😆 

oh and plz always reference the interface everywhere except in the dependency map (or unit test setup)

Reply:
😖 good catch! And ya, I'll try to reference the loader interface instead